<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ko≈Ço Fortuny</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="CustomOpenGlWidget" name="customOpenGlWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionWybierz_warianty"/>
    <addaction name="actionLosuj"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWybierz_warianty">
   <property name="text">
    <string>Wybierz warianty</string>
   </property>
  </action>
  <action name="actionLosuj">
   <property name="text">
    <string>Losuj!</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomOpenGlWidget</class>
   <extends>QWidget</extends>
   <header>customopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
